packages/gatsby-transformer-sqip/README.md

```diff
@@ -0,0 +1,33 @@
+# Gatsby SQIP plugin
+
+Generates vectorized primitive version of images to be used as preview thumbnails.
+
+## :hand: Usage
+
+### GraphQL
+```graphql
+image {
+  sqip(numberOfPrimitives: 3, blur: 0),
+  resolutions {
+    ...GatsbyContentfulResolutions_withWebp_noBase64
+  }
+}
+```
+
+### React
+
+#### Pure HTML
+
+Coming soon. Doing some preparations first.
```

Comment:
What are you intending here?

Reply:
Forgot about it, will add it later. It will work like this:
https://github.com/axe312ger/svg-loading-animation/blob/master/templates/mixins/image.pug#L5-L7
+
https://github.com/axe312ger/svg-loading-animation/blob/master/templates/style.css#L29-L45

Reply:
👍